src/Files.App/Filesystem/Security/AccessControlEntry.cs

```diff
@@ -30,10 +30,17 @@ public AccessControlType AccessControlType
 			}
 		}
 
+		public string AccessControlTypeHumanized
+			=> AccessControlType switch
+			{
+				AccessControlType.Allow => "Allow",
+				_ => "Deny" // AccessControlType.Deny
+			};
+
 		public string AccessControlTypeGlyph
 			=> AccessControlType switch
 			{
-				AccessControlType.Allow => "\xF13E",
+				AccessControlType.Allow => "\xE73E",
```

Comment:
Can you share the before and after?

Reply:
Old|New
---|---
![image](https://user-images.githubusercontent.com/62196528/228114827-77dc9ddf-e8f9-4f92-b563-2c7567e82824.png)|![image](https://user-images.githubusercontent.com/62196528/228114955-505890d3-dbba-447d-8c35-5ac864dbd6fa.png)

Previous one was smaller.